__tests__/app.test.js

```diff
@@ -99,11 +99,101 @@ test('200: Should return objects without body property', () => {
 		.get('/api/articles')
 		.expect(200)
 		.then(({ body }) => {
-			body.forEach((article) => {
+			body.articles.forEach((article) => {
 				expect(article).not.toHaveProperty('body');
 			});
 		});
 });
+describe('GET api/articles?topic=cats / mitch', () => {
+	test('200: Should return all articles that have cats as topic', () => {
+		return request(app)
+			.get('/api/articles?topic=cats')
+			.expect(200)
+			.then(({ body }) => {
+				body.articles.forEach((article) => {
+					expect(article.topic).toBe('cats');
+				});
+			});
+	});
+	test('200: Should return all articles that have mitch as topic', () => {
+		return request(app)
+			.get('/api/articles?topic=mitch')
+			.expect(200)
+			.then(({ body }) => {
+				body.articles.forEach((article) => {
+					expect(article.topic).toBe('mitch');
+				});
+			});
+	});
+	test('400: returns if invalid topic is passed', () => {
```

Comment:
Really good thinking here to have an error return with an invalid query. But if we think about this in the real world, if I was to make a search on say google for something that doesn't exist would I expect to see a bad request? I technically haven't done anything wrong as a user here, it just so happens that there are no articles available by my search criteria. There's two potential scenarios we could think of here:
- A topic that exists but doesn't have any articles related to it, this would be a 200 with an empty  response array.
- A topic that doesn't exist, would be a 404 not found.